src/main/java/com/github/manolo8/darkbot/utils/DiscordWebhook.java

```diff
@@ -0,0 +1,412 @@
+package com.github.manolo8.darkbot.utils;
+
+import javax.net.ssl.HttpsURLConnection;
+import java.awt.Color;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.reflect.Array;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ Example usage:
+
+ DiscordWebhook webhook = new DiscordWebhook("https://discordapp.com/api/webhooks/...");
+ webhook.setContent("Any message!");
+ webhook.setAvatarUrl("https://your.awesome/image.png");
+ webhook.setUsername("Custom Usernames!");
+ webhook.setTts(true);
+ webhook.addEmbed(new DiscordWebhook.EmbedObject()
+ .setTitle("Title")
+ .setDescription("This is a description")
+ .setColor(Color.RED)
+ .addField("1st Field", "Inline", true)
+ .addField("2nd Field", "Inline", true)
+ .addField("3rd Field", "No-Inline", false)
+ .setThumbnail("https://kryptongta.com/images/kryptonlogo.png")
+ .setFooter("Footer text", "https://kryptongta.com/images/kryptonlogodark.png")
+ .setImage("https://kryptongta.com/images/kryptontitle2.png")
+ .setAuthor("Author Name", "https://kryptongta.com", "https://kryptongta.com/images/kryptonlogowide.png")
+ .setUrl("https://kryptongta.com"));
+ webhook.addEmbed(new DiscordWebhook.EmbedObject()
+ .setDescription("Just another added embed object!"));
+ webhook.execute(); //Handle exception
```

Comment:
```suggestion
    Source: https://gist.github.com/k3kdude/fba6f6b37594eae3d6f9475330733bdb
```

Just properly link-back to the original source, if someone wants examples they can get them  there